i18n/en-US.yml

```diff
@@ -227,6 +227,7 @@ components:
     modeSelectorLabel: Select a transit mode
   BatchSettings:
     destination: destination
+    invalidModeSelection: Cannot plan a trip using the selected modes.
```

Comment:
Could we add another sentence like "Try including transit in your mode selection" just to point people in the right direction?

Reply:
Good idea! Added in https://github.com/opentripplanner/otp-react-redux/pull/990/commits/c72cbc70eeb78434188d524794b78dd25c406370.